docs/pages/api/wrap.md

```diff
@@ -1,3 +1,2 @@
-::: sklearn_raster.wrap
-
-::: sklearn_raster.FeatureArrayEstimator
\ No newline at end of file
+::: sklearn_raster.FeatureArrayEstimator
+::: sklearn_raster.estimator.FeatureArrayEstimator
```

Comment:
Is this possibly a copy-and-paste mistake (essentially duplicating the documentation for `FeatureArrayEstimator`)?  Not too big of an issue given that `wrap` no longer shows in the menu.